admin/src/scenes/settings/CleSettings.tsx

```diff
@@ -0,0 +1,251 @@
+import SimpleToggle from "@/components/ui/forms/dateForm/SimpleToggle";
+import React from "react";
+import { Container } from "@snu/ds/admin";
+import { CohortDto } from "snu-lib/src/dto";
+import ToggleDate from "@/components/ui/forms/dateForm/ToggleDate";
+
+type CleSettingsData = Partial<CohortDto>;
+
+type CleSettingsProps = {
+  cleSettingsData?: CleSettingsData;
+  isLoading: boolean;
+  readOnly: boolean;
+  handleChange: (data: CleSettingsData) => void;
```

Comment:
```suggestion
  onChange: (data: CleSettingsData) => void;
```
le `handleChange` est le nom de la fonction du parent qui gère vraiment le changement

Reply:
oui tu as raison, j'ai inversé, je corrige